### ISSUE TITLE
performance issues

### ISSUE THREAD

**Issue by Milesld** [B05901022/VOCANO#2]
Hello, I recently test your open source model with dataset ismir2014. But the onset-pitch/onset-offset-pitch performance can't achieve to the same as your paper. Is the open source model your latest model? Or did I do this right?
And I test the performance after I trained a new model with tonas and mir-1k datasets with your training code, the performance couldn't achieve the same as your paper too.

**Comment by Milesld** [comment]
And there is a strange phenomenon. I trained two models with your training code, one was supervised and the other was semi-supervised. But after I test the two models, the performance of supervised is better than the semi-supervised. Is there something wrong with my procedure?

**Comment by B05901022** [comment]
Hi, have you smoothen your labels before training? I made a small example [here][SDT_GEN], which onset/offset/duration flags are true on the neighboring frames (a total of 5 frames including the groundtruth ). This is also why we added a temporal decoding procedure to cancel out the smoothing effect.
For supervised and semi-supervised performance, we also found that selecting a good/similar unlabeled dataset might be crucial. If trained with monophonic data but semi-supervised learned with polyphonic data that is not clean enough, the performance might not be better. Also note that we performed source separation for polyphonic data, since we found that training with a monophonic dataset and semi-supervised training with a polyphonic dataset can not converge well.

[SDT_GEN]: https://github.com/B05901022/Note-Segmentation-SSL/issues/1
